pkg/operation/botanist/component/vpnshoot/vpnshoot.go

```diff
@@ -194,14 +179,11 @@ func (v *vpnShoot) Deploy(ctx context.Context) error {
 		if err != nil {
 			return err
 		}
-		mountPath := volumeMountPathSecret
-		if !v.values.ReversedVPN.Enabled {
-			mountPath = volumeMountPathSecretShoot
```

Comment:
Can we now drop https://github.com/gardener/gardener/blob/55ffc4bac04a2e068a5d87bbbf4c5cdff287bfba/pkg/operation/botanist/component/vpnshoot/vpnshoot.go#L73?